### PR TITLE
lookup table size fix

### PR DESCRIPTION
**Pull request by seiriosPlus**

### CODE REVIEW

**Comment by jacquesqiao** [comment]
Please add unit test:
https://github.com/PaddlePaddle/Paddle/blob/83c85f34e84d5bae22d374374408de780d10ae21/python/paddle/fluid/tests/unittests/test_dist_transpiler.py#L468
https://github.com/PaddlePaddle/Paddle/blob/83c85f34e84d5bae22d374374408de780d10ae21/python/paddle/fluid/tests/unittests/test_dist_transpiler.py#L500

File: python/paddle/fluid/transpiler/distribute_transpiler.py [hunk 1]
```diff
@@ -877,9 +877,15 @@ def _create_table_optimize_block(self, pserver_index, pserver_program,
         # create table param and grad var in pserver program
         origin_param_var = self.origin_program.global_block().vars[
             self.table_name]
+
+        zero_dim = long(
```
Comment (reyoung) [inline python/paddle/fluid/transpiler/distribute_transpiler.py]: do not use long since it is not compatible with `py3`

Comment (reyoung) [reply python/paddle/fluid/transpiler/distribute_transpiler.py]:
```python
print len(bin(sys.maxint))-2
```

In Python, int is 64bit.

Comment (seiriosPlus) [reply python/paddle/fluid/transpiler/distribute_transpiler.py]: Thanks for reminding.

**Review by jacquesqiao — APPROVED** [review]
LGTM!!~

**Pull request merged**